tools/etc/frr/support_bundle_commands.conf

```diff
@@ -47,7 +47,6 @@ show route-map
 show memory
 show interface vrf all
 show vrf
-show zebra fpm stats
```

Comment:
Do we know why this command is failing ? Is this command removed in the latest version ?

Reply:
it's never been implemented @srimohans as I recall we discussed this and you were going to submit the internal vmware work for it.

Reply:
I take it back it was there:
```/*
 * show_zebra_fpm_stats
 */
DEFUN (show_zebra_fpm_stats,
       show_zebra_fpm_stats_cmd,
       "show zebra fpm stats",
       SHOW_STR
       ZEBRA_STR
       "Forwarding Path Manager information\n"
       "Statistics\n")
{
        zfpm_show_stats(vty);
        return CMD_SUCCESS;
}

```

Reply:
ah the problem is that if you do not have fpm turned on as a module the command will not work.

Reply:
We use this command quite often in our debugging and adding more stuff to this command recently. I would say, let's leave this command in support bundle as-is and I will check if we can just return success from this CLI in the case when FPM is not running.